test/sagemaker_tests/mxnet/inference/conftest.py

```diff
@@ -119,6 +120,9 @@ def instance_type(request, processor):
 def accelerator_type(request):
     return request.config.getoption('--accelerator-type')
 
+@pytest.fixture(scope='session', name='multi_region_support')
+def fixture_multi_region(request):
```

Comment:
Do we need accompanying changes in `mxnet/inference/integration/sagemaker` tests to make use of this fixture?

Reply:
Reverting the change as the feature is been added for training tests on this PR.